homeassistant/components/nuki/lock.py

```diff
@@ -65,23 +67,22 @@ class NukiDeviceEntity(NukiEntity, LockEntity, ABC):
     _attr_supported_features = LockEntityFeature.OPEN
 
     @property
-    def name(self):
+    def name(self) -> str | None:
         """Return the name of the lock."""
         return self._nuki_device.name
 
     @property
-    def unique_id(self) -> str:
+    def unique_id(self) -> str | None:
         """Return a unique ID."""
         return self._nuki_device.nuki_id
 
     @property
-    def extra_state_attributes(self):
+    def extra_state_attributes(self) -> dict[str, Any]:
         """Return the device specific state attributes."""
-        data = {
+        return {
```

Comment:
This is the only change that is not a type-hint adjustment, but it's a very small code cleanup.